spec/dummy/config/secrets.yml

```diff
@@ -0,0 +1,6 @@
+
+development:
+  secret_key_base: b9ed8530bded47950895f26188cc21a984e271e262d8e8b8c78c8fc57c65a65fd5818db617906414b9f8b5f608bdc115af3aaa1714431e66aa538c2b8c749304
```

Comment:
Why do you need this?

Reply:
It's needed for versions 5.0 and 5.1 of rails